Journalbeat 7.9.0-SNAPSHOT panics on start

I'm seeing Journalbeat consistently fail during startup with the below.

```
2020-07-21T11:27:00.200Z        INFO    [journalbeat]   beater/journalbeat.go:93        journalbeat is running! Hit CTRL-C to stop it.
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x16ee79c]

goroutine 79 [running]:
github.com/elastic/beats/v7/journalbeat/pkg/journalread.(*Reader).Next(0x0, 0x7fc38237c200, 0xc000103920, 0x7fc38237c200, 0xc000103920, 0x0)
        /go/src/github.com/elastic/beats/journalbeat/pkg/journalread/reader.go:155 +0x4c
github.com/elastic/beats/v7/journalbeat/reader.(*Reader).Next(0xc0000d26c0, 0xc000103920, 0x0, 0x0)
        /go/src/github.com/elastic/beats/journalbeat/reader/journal.go:108 +0x75
github.com/elastic/beats/v7/journalbeat/input.(*Input).publishAll.func1(0xc0002702b0, 0xc0000d49a0, 0xc0000d26c0, 0xc000102720)
        /go/src/github.com/elastic/beats/journalbeat/input/input.go:168 +0xa6
created by github.com/elastic/beats/v7/journalbeat/input.(*Input).publishAll
        /go/src/github.com/elastic/beats/journalbeat/input/input.go:158 +0x148
```

- Version: `7.9.0-SNAPSHOT`, `2020-07-21T11:27:00.002Z        INFO    [beat]  instance/beat.go:985    Build info      {"system_info": {"build": {"commit": "f23ce6145d301465c29cca0c94135db21946d234", "libbeat": "7.9.0", "time": "2020-07-20T02:40:44.000Z", "version": "7.9.0"}}}`
- Operating System: ran on `Container-Optimized OS`, Kernel version `4.19.112+` in Kubernetes Pod
- Steps to Reproduce: run Journalbeat with the below config
```
journalbeat:
  inputs:
  - cursor_seek_fallback: tail
    paths: null
    seek: cursor
output:
  elasticsearch:
    hosts:
    - https://elasticsearch-es-http.default.svc:9200
    password: redacted
    ssl:
      certificate_authorities:
      - /mnt/elastic-internal/elasticsearch-certs/ca.crt
    username: default-journalbeat-beat-user
processors:
- add_cloud_metadata: null
- add_host_metadata: null
```

--- a/journalbeat/reader/journal.go
+++ b/journalbeat/reader/journal.go
@@ -76,6 +76,7 @@ func newReader(path string, c Config, done chan struct{}, state checkpoint.Journ
 	instance.AddJournalToMonitor(c.Path, journal)
 
 	return &Reader{
+		r:       r,
 		journal: journal,
 		config:  c,
 		done:    done,

